target/linux/ath79/image/generic.mk

```diff
@@ -151,6 +152,12 @@ define Build/wrgg-pad-rootfs
 	$(STAGING_DIR_HOST)/bin/padjffs2 $(IMAGE_ROOTFS) -c 64 >>$@
 endef
 
+define Build/zyxel-tar-bz2
+	mv $@ $(word 2,$(1))
```

Comment:
Please don't just use static file names (if I read this correctly), but use some $@.tmp folder where you can put them ...

Reply:
While the exact file names to be put into the tar archive need to match what the updater is expecting, the tar operation itself may as well be executed on files in a temp folder (i.e. create a new folder, move files there, call tar and `rm -rd` the folder). I'll add that tomorrow.

Reply:
good example here, if you need it

target/linux/ath79/image/common-senao.mk